Gemfile

```diff
@@ -35,6 +36,7 @@ gem "byebug",                                          :require => false
 gem "color",                          "~>1.8"
 gem "config",                         "~>1.6.0",       :require => false
 gem "dalli",                          "=2.7.6",        :require => false
+gem "dbus-systemd",                   "~>1.1.0",       :require => false
```

Comment:
We may want to bucket systemd based stuff into a bundler group, as technically this stuff is not need on non-systemd systems (macs, dev envs using Procfile, containers, etc)

Reply:
:+1: there's an optional group for the systemd journald logger gem, I'll move this down there